synapse/storage/databases/main/events_worker.py

```diff
@@ -130,6 +129,15 @@ def __init__(self, database: DatabasePool, db_conn, hs):
                     db_conn, "events", "stream_ordering", step=-1
                 )
 
+        if not hs.config.worker.worker_app:
```

Comment:
Can probably be put on the background worker?

Reply:
Ugh, yes but that work isn't in 1.21 :(

Reply:
Oh, right. 🤦 I'll update it when this gets merged forward.

Reply:
So if we're retargetting this to develop, seems like we can make this change!